### PR TITLE
Update for new electrical

### PR DESCRIPTION
**Pull request by BigBroccoli**
Tested and works, if there are changes it'll be tuning later

### CODE REVIEW

File: MoonDawgArduino/Libraries/Movement/Movement.h [hunk 1]
```diff
@@ -52,15 +53,10 @@ class DriveMovement{
 private:
     Servo motor1;
```
[the remark this answers is not in the document]
Comment (abarnes6) [reply MoonDawgArduino/Libraries/Movement/Movement.h:54]: also same thing applies to the classes above this one

File: MoonDawgArduino/Libraries/Movement/Movement.cpp [hunk 2]
```diff
@@ -12,64 +12,63 @@ DigMovement::DigMovement() {
...
+void DigMovement::digSetup(uint8_t diggingPin, uint8_t actuatorPin) {
+    motor1.attach(diggingPin);
+    motor2.attach(actuatorPin);
+    motor1.write(90);
+    motor2.write(90);
 }
 
-void DigMovement::digMotorBackward(){
-    motor.write(0);
+void DigMovement::digMotor(uint8_t speed){
```
Comment (abarnes6) [inline MoonDawgArduino/Libraries/Movement/Movement.cpp:22]: maybe call this something more specific, like digMotorStart()

Comment (abarnes6) [reply MoonDawgArduino/Libraries/Movement/Movement.cpp:22]: action words for functions >>

File: MoonDawgArduino/Libraries/Movement/Movement.h (hunk 1 above)
[the remark this answers is not in the document]
Comment (BigBroccoli) [reply MoonDawgArduino/Libraries/Movement/Movement.h:54]: They can be. But don't really need to be as they are being described by the class they are in.

File: MoonDawgArduino/Libraries/Movement/Movement.cpp (hunk 2 above)
Comment (BigBroccoli) [reply MoonDawgArduino/Libraries/Movement/Movement.cpp]: done

File: MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino [hunk 3]
```diff
@@ -1,146 +1,137 @@
...
+
+    driving.driveSetup(LEFT_MOTOR, RIGHT_MOTOR);
+    depositing.depositSetup(DEPOSIT_MOTOR, DEPOSIT_VIBRATOR);
+    digging.digSetup(DIGGING_MOTOR, DIGGING_ACTUATOR);
+}
+
+void loop() {
+  // put your main code here, to run repeatedly
+
+    if (Serial.available() > 0) {
```
Comment (abarnes6) [inline MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino:115]: should move this serial read block into its own function, returning the char array and passing it to command processing

**Review by abarnes6 — APPROVED** [review]

File: MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino (hunk 3 above)
Comment (BigBroccoli) [reply MoonDawgArduino/MoonDawgArduino/MoonDawgArduino.ino:115]: Let's note this and look at it in the new semester so we can test it still works correctly on the bot.

**Pull request merged**